auth/oidc.go

```diff
@@ -325,12 +325,12 @@ func (op *OIDCProvider) initOIDCClient(ctx context.Context) error {
 		return err
 	}
 
-	metadata, verifier, err := op.DiscoverConfig(ctx)
+	metadata, _, err := op.DiscoverConfig(ctx)
```

Comment:
It doesn't look like this leaves any consumers for the verifier returned by DiscoverConfig.  I'm not convinced this was an intentional change.  I realize this PR doesn't change that, but I feel this might merit a tracking ticket to evaluate whether we should be using the verifier returned by DiscoverConfig in the case where we're using standard discovery.

Reply:
https://issues.couchbase.com/browse/CBG-2752